package.json

```diff
@@ -1,22 +1,21 @@
 {
   "dependencies": {
+    "@scalar-labs/scalardl-javascript-sdk-base": "^1.0.0",
```

Comment:
I think it's now updated to 1.1.0 from the license change (?)

Reply:
In package.json, `^1.0.0` is OK because it means
```
^1.2.3 := >=1.2.3 <2.0.0
 ```
Reference: https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

However, we have to update yarn.lock files in web/node after we update scalardl-javascript-sdk-based in NPM registry since they stick to @scalar-labs/scalardl-javascript-sdk-base 1.0.0 now.

yarn.lock
```
  "@scalar-labs/scalardl-javascript-sdk-base@^1.0.0":
    resolved "https://registry.yarnpkg.com/@scalar-labs/scalardl-javascript-sdk-base/-/scalardl-javascript-sdk-base-1.0.0.tgz#f0b00560a01d438f44c50d1a71f68f8cf844a7c4"
```